tests/local_mode/test_language.py

```diff
@@ -49,8 +53,12 @@ def test_vector_non_valid_token_is_zero():
     """Test that the vector of non valid token is all zeros"""
 
     doc = nlp("outofvocabularytoken")
-    actual = doc[0].vector
-    zeros = np.zeros(actual.shape)
+    # Convert numpy vector from doc var to torch tensor
```

Comment:
same as above

Reply:
Thanks Nilansh, I did remove the conversion for the vector coming from doc object, but to specify zeros tensor I wanted to know do we need to specify or know the data type of doc vector or is it going to be float32 ?

Reply:
Also, from your commend below I understand that I can specify the datatype within the getitem function in vectors.py (where I am supposed to convert it) to resolve this, does this work ?

Reply:
check vectors.py, the vector loaded there is an np array, but you need to convert it to torch tensor, check __getitem__ method in vectors.py file,